src/manifest/MsQuicEtw.man

```diff
@@ -199,41 +199,49 @@
                 value="2"
                 />
             <map
-                message="$(string.Enum.QUIC_API_TYPE.STRM_CLOSE)"
+                message="$(string.Enum.QUIC_API_TYPE.SET_CONFIGURATION)"
```

Comment:
This isn't going to break decoding old logs, right?

Reply:
Kind of, but we don't have a choice because the enum in the code changed. I'd rather be correct with main.

Reply:
Ok

Reply:
Then we should backport this to Ni.

Reply:
Ok.